packages/framework/presence/src/presence.ts

```diff
@@ -60,8 +70,18 @@ export interface ISessionClient<
 	 *
 	 * @remarks
 	 * Connection id will change on reconnect.
+	 *
+	 * If {@link ISessionClient.status} is {@link SessionClientStatus.Disconnected}, this will represent the last known connection id,
```

Comment:
```suggestion
	 * If {@link ISessionClient.status} is {@link SessionClientStatus.Disconnected}, this will represent the last known connection id.
```

Reply:
...or if this client (self) is disconnected too. Be sure to incorporate that.